globus_action_provider_tools/flask/config.py

```diff
@@ -0,0 +1,11 @@
+import dataclasses
+
+
+@dataclasses.dataclass(frozen=True)
+class ActionProviderConfig:
+    # When enabled, error messages returned to users will not include any user provided
+    #   data.
+    validation_error_obscuring: bool = True
```

Comment:
Can you help me understand why you'd prefer an active verb?

I was shooting for a referenceable feature name given that this is a feature we're exposing in a public library. `validation_error_obscuring` is more obvious as a feature name to me, a thing you'd enable/disable in a declarative config like this, since it's (I think) a noun phrase.
  * "Please enable 'ValidationErrorObscuring'"

`obscure_validation_errors` on the other hand is a full declaration
  * "Please enable 'ObscureValidationErrors'"

---

`full_validation_error_messages` does read like a feature ("Please enable 'FullValidationErrorMessages'") but doesn't adequately represent what we're doing here. We're not returning a partial error message when that's disabled, we're returning an obfuscated one (we replace it with a generic `"NOPE"` string).

Reply:
Oh I just understood the part about obscure being an adjective.

I don't hate `obscured_validation_errors` or even `scrubbed_validation_errors` or something.

Reply:
I think if I see "obscure" I'm reflexively going to disable it without asking why I would want that on. Could we instead move toward something like "prevent_input_reflection_on_validation_error" with a warning in the comment that this means JSON Schema errors will just emit a generic message?

Reply:
`prevent_input_reflection_on_validation_error` is very close to what I initially had (`include_user_data_in_validation_errors` or something similar) and I changed it because it felt so wordy as a feature to expose.

Reply:
I think we generally don't want people changing it, it's an escape hatch, so personally the wordiness doesn't bother me too much? But open to differing opinions. I'd prefer the precision at the cost of wordiness.

Reply:
Discussed in person—alongside some improvements to the default error message (including the path and validator) we agreed that "scrubbed" is a pretty good compromise.

Reply:
Replaced w/ `scrubbed_validation_errors`.